app/controllers/metadata/MetadataController.scala

```diff
@@ -19,7 +19,7 @@ package controllers.metadata
 import connectors.{DataStoreConnector, EmailThrottlerConnector}
 import domain.SDESInputFormats._
 import domain.{Notification, NotificationsForEori}
-import models.{EmailTemplate, HistoricDocumentRequestSearch, SearchResultStatus}
+import models.{EORI, EmailTemplate, HistoricDocumentRequestSearch, SearchResultStatus}
```

Comment:
You have a unused import here

[warn] customs-financials-api/app/controllers/metadata/MetadataController.scala:22:16: Unused import
[warn] import models.{EORI, EmailTemplate, HistoricDocumentRequestSearch, SearchResultStatus}

Reply:
Done the required changes